packages/std/src/traits.rs

```diff
@@ -246,3 +250,29 @@ impl<'a> QuerierWrapper<'a> {
         Ok(res.delegation)
     }
 }
+
+#[cfg(test)]
+mod test {
```

Comment:
FYI: https://github.com/CosmWasm/cosmwasm/issues/653
But no need to change here. We change it for the whole codebase anyways.